website/cue/reference.cue

```diff
@@ -332,7 +332,7 @@ _values: {
 		// `common` specifes that the option is commonly used. It will bring the
 		// option to the top of the documents, surfacing it from other
 		// less common, options.
-		common: bool
+		common?: bool
```

Comment:
We don't emit any of the `common` stuff in the machine-generated documentation, so this has to be an optional field.